apache/rocketmq/v2/service.proto

```diff
@@ -193,6 +193,10 @@ message Publishing {
   // reject the request. As a result, it is advisable that Producer performs
   // client-side check validation.
   int32 max_body_size = 3;
+
+  // When `check_message_type` flag set `false`, no need to check message's type
+  // with messageQueue's `accept_message_types` before publising.
+  bool check_message_type = 4;
```

Comment:
It's generally discouraged to use bool in the protocol definition. Prefer explicit enum.
For example
enum ValidateMessageTypeStrategy {
 UNSPECIFIED = 0,
DISABLED = 1,
PERMISSIVE = 2,
ENFORCING = 3,
}